src/content/components/Triage/Triage.js

```diff
@@ -116,6 +116,65 @@ export class Triage extends React.PureComponent {
     this._isMounted = false;
   }
 
+  async fetchBugs() {
+    const prevIteration = this.context.iterations.getAdjacentIteration(-1)
+      .number;
+    const result = await this.context.qm.runQueries([
```

Comment:
If all these queries are the same as they are in `componentWillMount`, then you should try to deduplicate them. Move the queries into a class method called `getQueries` that just returns the array that is passed to runQueries here. Then all the methods that use the queries (fetchBugs and componentWillMount) can just do `this.context.qm.runQueries(this.getQueries())`